databuilder/main.py

```diff
@@ -146,7 +146,7 @@ def load_definition(definition_file):
         raise AttributeError("A dataset definition must define one 'dataset'")
     assert isinstance(
         dataset, Dataset
-    ), "'dataset' must be an instance of databuilder.query_language.Dataset()"
+    ), "'dataset' must be an instance of databuilder.ehrql.Dataset()"
```

Comment:
Is there anything we can do to stop imports from the `query_language` module?  Would doing so gain us anything?

Reply:
I'm not sure there is and I'm not sure it would. As long as the documented modules are clear I don't think there's much to be gained from trying to stop people doing silly things which they have to go out of their way to do.

Reply:
Fine.